program-analysis/echidna/exercises/exerciseX/NaughtCoin.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {ERC20} from "./level-utils/ERC20.sol";
+
+contract NaughtCoin is ERC20 {
+    // string public constant name = 'NaughtCoin';
+    // string public constant symbol = '0x0' -> 'NTC';
+    // uint public constant decimals = 18;
+    uint256 public timeLock = block.timestamp + 10 * 365 days;
+    uint256 public INITIAL_SUPPLY;
+    address public player;
+
+    // Note that the original symbol is changed from 0x0 to NTC
+    // This is caused by https://github.com/crytic/echidna/issues/909
+    constructor(address _player) ERC20("NaughtCoin", "NTC") {
```

Comment:
The `ERC20` symbol had to be changed because of https://github.com/crytic/echidna/issues/909

Reply:
Oh, this is fine, thanks for documenting this.